ext_emconf.php

```diff
@@ -13,10 +13,10 @@
     'state' => 'stable',
     'uploadfolder' => 0,
     'clearCacheOnLoad' => 0,
-    'version' => '1.7.3',
+    'version' => '2.0.0-dev',
```

Comment:
I assume you'll do a release commit later to remove the -dev suffix?

Reply:
True.. :)